### DISCUSSION TITLE
Displaying external MIDI using Web MIDI API

### DISCUSSION THREAD

**Discussion by oscarhermoso** [dannywritescode/svelte-piano#4] [Show and tell]
Hey @danferns,

Love this repo, I forked and took it a step further to display MIDI input from an external device.

https://github.com/oscarhermoso/svelte-piano

https://oscarhermoso.github.io/svelte-piano/

Would have been nice to do it all inside of the Svelte REPL, but their iframe doesn't have permissions enabled for requesting MIDI access.

**Comment by danferns** [comment]
Hey Oscar,

Thank you for sharing your fork, it is amazing! It works very well, and the notes fading out over time makes it feel so much more lively.

I think this is a more complete demo of the piano. I will link to this in the README!
